modules/social_features/social_user/social_user.module

```diff
@@ -754,3 +755,15 @@ function social_user_views_data_alter(array &$data): void {
     ];
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Override the default drupal truncate function for all usernames, so the whole
+ * name will be displayed.
+ */
+function social_user_preprocess_username(array &$variables): void {
```

Comment:
This is something that can affect our community users. We need to make sure to add to the PR an explanation of what is changing with examples of before and after. This will be super important for the transparency of our release :D 

Thank you very much!

Reply:
I think that it will not affect anything because this code was simply moved from the theme (see related theme MR). Also, I'd say that an additional explanation of changes is not needed because it should be clear from the "How to test" section or not? And the last one, I'd really appreciate it if you give us the possibility to review PRs by our team at first and start reviewing from your side only when our review is finished.

Reply:
Hi @OH17L , thank you for your reply and clarifications. 

> I'd really appreciate it if you give us the possibility to review PRs by our team at first and start reviewing from your side only when our review is finished.

I understood that this PR was already done, this is why I added my comments here. Please let me know when I can review it.